### PR TITLE
[DB-6992, DB-7006] Updates to help message for import data file

### PR DESCRIPTION
**Pull request by priyanshi-yb**
1. removed the --table-list and --exclude-table-list from help msg of import data file (fixes - [DB-6992](https://yugabyte.atlassian.net/browse/DB-6992))

2. enhanced the description of the --data-dir flag for the cloud storage Urls (fixes - [DB-7006](https://yugabyte.atlassian.net/browse/DB-7006))

3. marked the --file-table-map flag as required.

### CODE REVIEW

**Review by shubham-yb — COMMENTED** [review]
Can we add the cloud support in the parent description as well? Something like:
`This command imports data from given files into YugabyteDB database. The files can be present either in local directories for cloud storages like AWS, GCS and Azure.`

**Review by shubham-yb — COMMENTED** [review]
Can we change the help message from the `--start-clean` flag since it's not applicable anymore. We can add separate messages for both commands like:
```
import-schema: deletes all the existing schema objects before importing the schema.
import-data: starts a fresh (empty table) or incremental data load
```

File: yb-voyager/cmd/importDataFileCommand.go [hunk 1]
```diff
@@ -381,16 +381,24 @@ func init() {
 		`character used as delimiter in rows of the table(s)(default is comma for CSV and tab for TEXT format)`)
 
 	importDataFileCmd.Flags().StringVar(&dataDir, "data-dir", "",
-		"path to the directory contains data files to import into table(s)")
+		"path to the directory contains data files to import into table(s)\n"+
```
Comment (shubham-yb) [inline yb-voyager/cmd/importDataFileCommand.go:384]: `path to the directory which contains the data files to be imported into the table(s)`

**Review by shubham-yb — COMMENTED** [review]
Can we remove `Note: default will be to import all the files in --data-dir with given format, for eg: table1.csv to import into table1 on target database.` from file-table-map

**Review by amit-yb — APPROVED** [review]
LGTM. 👍

**Review by shubham-yb — APPROVED** [review]

**Pull request merged**